### DISCUSSION TITLE
Collection Hidden by Default

### DISCUSSION THREAD

**Discussion by mdurao** [phantom/docs#115] [Help & Support] [answered]
### Summary

I've created a new Collection of cNFTs and when I transfered one cNFT to my Phantom wallet, the collection gets hidden by default.

How can I prevent this?

Thank you in advance,

Miguel Durão (aka: Kamafeu from Discord)

### Steps to Reproduce

I can send you one cNFT if needed, just provide me a wallet.

### Phantom Version

23.15.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hi @mdurao,

We use a number of factors to determine whether or not a collection is hidden by default in order to protect our users from malicious activity and spam. If you would like to provide your collection address and more information about your project I would be happy to have it reviewed.

**Comment by mdurao** [comment]
Hi @adamdelphantom 

Sure thing, the collection address is: `RRCLBFNZ2wH1bwrhNx8pgEkQrpW5qpUDGDSvDeJnWE4`.

You can get more info about the collection and the project here: https://solana-bear-gang.gitbook.io/whitepaper/rumble-racers-pvp-game/rr-what-is-it

Thank you,

Miguel
